### ISSUE TITLE
Mods do not stay disabled when included in an exported profile.

### ISSUE THREAD

**Issue by Rensik** [gorilla-devs/GDLauncher#649]
**Describe the bug**

When a mod that is disabled is included in the export of a profile, when importing said profile elsewhere, the mod is no longer disabled.

**Expected behavior**

The mod should remain disabled when included in an exported profile..

**Comment by RemagOfficial** [comment]
please fill in the whole report or delete unused parts

**Comment by RemagOfficial** [comment]
please fill in the whole report or delete unused parts

**Comment by Rensik** [comment]
Edited

**Comment by CvoidWyvern** [comment]
I've also had a lot of trouble with this while sending a wip modpack to others to help work on it, looks like the launcher just ignores if a mod is disabled or not when exporting.

**Comment by Ladvace** [comment]
I'll try to make a pr as soon as i can

**Comment by TOLoneWolf** [comment]
Twitch modpack format does not support disabled mods as they do not support "optional mods" in their system like ATLauncher. When you include it in a export it, it is expected to be used.

**Comment by Rensik** [comment]
And there's no way to hijack and check to see if a mod should be included or not?

**Comment by TOLoneWolf** [comment]
> check to see if a mod should be included or not?  

Yeah it's you checking the box next to export x files into the export. You told it to include it.

Now gdl could ad its own tracking for disabled mods but any users importing it into another launch would still have them active by default since curseforges system doesn't have that. Multimc, curselauncher, atl, etc.
